roles_n_rules/role_data.py

```diff
@@ -1,9 +1,114 @@
 night_users = [
-    ["Cupid","Dog","Look-Alike","Infected Wolf"],
+    ["Cupid","Dog","Look-Alike"],
     ["Assassin","Aura Teller","Exorcist","Fortune Teller","Hooker","Innkeeper","Priest","Priestess","Raven","Curse Caster",
-        "Lone Wolf","Warlock","Devil","Ice King","Pyromancer","The Thing","Vampire","Zombie"],
+        "Infected Wolf","Lone Wolf","Warlock","Devil","Ice King","Pyromancer","The Thing","Vampire","Zombie"],
     [],
     ["Crowd Seeker","Grandma","Tanner"]
     ]
 
-day_users = [["Barber","Royal Knight"],[],[],["Tanner"]]
+day_users = [[],["Barber","Royal Knight"],[],["Tanner"]]
+
+import management.db as db
+from management.db import db_get, db_set
+from main_classes import Mailbox
+from management.dynamic import lifepotion_in_play
+
+def death():
+    """What is death?  
+    In this case it means emptying death-row. Let's see what you got in stock for us!"""
+
+    answer = Mailbox()
+    deadies = []
+    if lifepotion_in_play():
+
+        return answer.log("At the end of the night, no deaths were found due to the witch's life potion.")
+
+    # [id,victim,role,murderer]
+    order = db.get_kill()
+
+    while order != None:
+
+        victim_id = int(order[1])
+        victim_role = db_get(victim_id,'role')
+        role = order[2]
+        murderer = order[3]
+
+        if victim_role not in ['Dead','Spectator','Suspended',None,'Unknown']:
+            # Standard day town kill
+            if role in ["Innocent"]:
+                pass
+
+            # Standard night town kill
+            if role in ["Assassin","Cult Leader","Witch"]:
+                pass
+
+            # Standard night wolf kill
+            if role in ["Werewolf","Lone Wolf","White Werewolf"]:
+                pass
+
+            # Instant kill
+            if role in ["Barber","Cupid","Executioner","Huntress","Devil","Zombie"]:
+                pass
+
+            # Add Priest test
+            if role in ["Priest"]:
+                pass
+            
+            # Devil's soul kill (not the wager!)
+            if role in ["Demon"]:
+                pass
+
+            # Horseman kill
+            if role in ["Horseman"]:
+                pass
```

Comment:
all those passes are todos i assume? if so please mark them with
```py
pass  # todo
```
so they are easier to find later on